code/game/turfs/closed/wall/window_frames.dm

```diff
@@ -193,43 +178,35 @@
 	else
 		. += "<span class='notice'>The window frame is empty</span>"
 
-/turf/closed/wall/window_frame/proc/on_painted(is_dark_color)
-	SIGNAL_HANDLER
-
-	if (is_dark_color) //Opaque directional windows restrict vision even in directions they are not placed in, please don't do this
-		set_opacity(255)
-	else
-		set_opacity(initial(opacity))
```

Comment:
This logic's still on the window itself ye?

Reply:
yes, this is actually a remnant from when i was trying to make windows into just the frame, with the actual glass part being the same object. but i gave up on that when i changed frames to turfs. so this does nothing